packages/dev/sharedUiComponents/src/components/reactGraphSystem/GraphConnectorHandle.tsx

```diff
@@ -0,0 +1,53 @@
+import type { FC } from "react";
+import { useCallback } from "react";
+import type { DropTargetMonitor } from "react-dnd";
+import { useDrag, useDrop } from "react-dnd";
+import { ClassNames } from "../classNames";
+import style from "./GraphConnectorHandle.modules.scss";
+import { useGraphContext } from "./useGraphContext";
+
+export interface IGraphConnectorHandlerProps {
+    parentId: string;
+    parentX: number;
+    parentY: number;
+    offsetX?: number; //offset relative to the center of the parent
+    offsetY?: number; //offset relative to the center of the parent
+    parentWidth: number;
+    parentHeight: number;
+    parentContainerId: string;
+}
+
+export const GraphConnectorHandler: FC<IGraphConnectorHandlerProps> = (props) => {
+    const { parentId, parentX, parentY, offsetX = 0, offsetY = 0, parentWidth, parentHeight, parentContainerId } = props;

```

Comment:
Very nit-picky - there is no need to destruct here. You can destruct in the function call:
```suggestion
export const GraphConnectorHandler: FC<IGraphConnectorHandlerProps> = ({ parentId, parentX, parentY, offsetX = 0, offsetY = 0, parentWidth, parentHeight, parentContainerId }) => {
```

It depends on the props type you defined, but it should work this way.